styles/mixins/spacing.js

```diff
@@ -14,8 +14,22 @@ module.exports = function (config) {
     return breakpointHelper(styles, config);
   };
 
-
+  const axials = {
```

Comment:
I know this goes against what I originally said, but I think this logic belongs in the function above, so you do the position check there. would that make things cleaner?